### PR TITLE
First deliverable

### PR DESCRIPTION
**Pull request by hamg26**
First deliverable
- Create an API
- Add an endpoint to read from a CSV file
- The CSV should have any information
- The items in the CSV must have an ID element (int value)
- The endpoint should get information from the CSV by some field ***(example: ID)***
- The result should be displayed as a response
- Clean architecture proposal
- Use best practices
- Handle the Errors ***(CSV not valid, error connection, etc)***

### How to run?
- Clone the repository in `$GOPATH/src/github.com/hamg26/academy-go-q42021`
- Run `go mod tidy`
- Run `go install`
- Run `$GOPATH/bin/academy-go-q42021`

### Endpoints:
- http://localhost:3031/pokemons image
- http://localhost:3031/pokemons/:id 
image

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+	CSV struct {
+		Path string
+	}
+	Server struct {
+		Address string
+	}
+	Logging bool
+}
+
+var C config
```
Comment (alvarocueva) [inline config/config.go:22]: By convention, all the exported stuff (variables, functions, etc) should have a description
```go 
// C - Description here
```

Please do this for all the exported stuff

File: config/config.go [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+func ReadConfig() {
```
Comment (alvarocueva) [inline config/config.go:24]: This is cool, good job!

Comment (hamg26) [reply config/config.go:24]: Thanks for your review!!

File: config/config.go (hunk 1 above)
Comment (hamg26) [reply config/config.go:22]: Are you OK with me adding those in the second deliverable branch? 
With the amount of changes in that other branch would mean adding a lot of comments for things that are going to disappear

Comment (hamg26) [reply config/config.go:22]: See the comments now in the SECOND-DELIVERABLE PR [here](https://github.com/hamg26/academy-go-q42021/pull/2/commits/94c4f3df1986e94dfa4774892c63189909479933)

**Pull request merged**